src/server/passport/strategies.js

```diff
@@ -0,0 +1,33 @@
+const responseManager = require('../routes/controllers/utilities/responseManager')
+
+const loginStrategy = (UserModel = require('../models/user'), strategyUtils = require('./strategyUtils')) => {
+	const validateUser = (username, password, done) => {
+		const index = {
+			username
+		}
+		
+		UserModel.findOne(index).exec()
+			.then(user => strategyUtils.validateUserPassword(user, password, done))
+			.catch(() => {
+				const userToSerialize = null
+				done(responseManager.MESSAGES.errors.UNAUTHORIZED, userToSerialize)
+			})
+	}
+
+	const serialzeUser = (user, done) => {
+		const error = null
+		done(error, user.id)
+	}
+
+	const deserializeUser = (id, done) => UserModel.findById(id, done)
```

Comment:
What if we won't find any?

Reply:
Then done() send to handler unauthorized error message.